lib/metrics/amplitude.js

```diff
@@ -0,0 +1,209 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// This module contains mappings from activity/flow event names to
+// amplitude event definitions. The intention is for the returned
+// `receiveEvent` function to be invoked for every event and the
+// mappings determine which of those will be transformed into an
+// amplitude event. You can read more about the amplitude event
+// structure here:
+//
+// https://amplitude.zendesk.com/hc/en-us/articles/204771828-HTTP-API
+
+'use strict'
+
+const APP_VERSION = /^[0-9]+\.([0-9]+)\./.exec(require('../../package.json').version)[1]
+
+const GROUPS = {
+  activity: 'fxa_activity',
+  email: 'fxa_email',
+  login: 'fxa_login',
+  registration: 'fxa_reg',
+  sms: 'fxa_sms'
+}
+
+const EVENTS = {
+  'account.confirmed': {
```

Comment:
nit: Should this be `account.login.confirmed` like `account.login.confirmedUnblockCode` below? I could see how this would be easily confused with an account sign-up verified.

Reply:
Nope, unless I'm parsing the [taxonomy document](https://docs.google.com/spreadsheets/d/1G_8OJGOxeWXdGJ1Ugmykk33Zsl-qAQL05CONSeD4Uz4/edit#gid=879531055) wrong. It defines this one as:

> User clicked on link in login confirmation email

And here are the two events together in the logs, occurring just after a sign-in confirmation email has been sent:

```
flowEvent {"event":"account.confirmed","locale":"en","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10.11; rv:56.0) Gecko/20100101 Firefox/56.0","time":1503729641792,"flow_id":"01e1f107fc50532944b4264f085f79cf55d9bbcbe3e3fbfaceff2475c6cb894e","flow_time":34418,"flowBeginTime":1503729607374,"flowCompleteSignal":"account.signed","flowType":"login","uid":"5160ebf3b6b94b6baa4c403d20bcb00e"}
amplitudeEvent {"time":1503729641792,"event_type":"fxa_login - email_confirmed","session_id":1503729607374,"event_properties":{"service":"sync"},"user_properties":{"flow_id":"01e1f107fc50532944b4264f085f79cf55d9bbcbe3e3fbfaceff2475c6cb894e","ua_browser":"Firefox","ua_version":"56","ua_os":"Mac OS X","fxa_uid":"5160ebf3b6b94b6baa4c403d20bcb00e"},"app_version":"94","language":"en"
```

Seems correct to me?